examples/tests/mixed_units.nbt

```diff
@@ -4,33 +4,50 @@
 assert_eq(38° + 53′ + 23″, 38.8897°, 1e-4°)
 assert_eq(-(77° + 0′ + 32″), -77.0089°, 1e-4°)
 
-assert_eq(38.8897° -> DMS, "38° 53′ 23″")
-assert_eq(-77.0089° -> DMS, "-77° 0′ 32″")
+assert_eq("{38.88972222° -> DMS}", "{[38°, 53′, 23″]}")
```

Comment:
Can you explain this (and other) changes below? What happens if we leave the left-hand side as it was?

Reply:
> Can you explain this (and other) changes below? What happens if we leave the left-hand side as it was?

Because the unitlist allows for decimals on the last element, unlike the previous string implementation, the result ends up being a decimal number. I changed the left hand side to compensate for this, but I could also change the right hand side instead, if you think that would be better. 
Alternatively we could also round the last element, though that would mean that the operation wouldn't be reversible as it is now through addition.

Reply:
> I changed the left hand side to compensate for this, but I could also change the right hand side instead, if you think that would be better.

I think that would be better because we could clearly check in the diff if there are any changes to the previous behavior. I would like that.

> Alternatively we could also round the last element, though that would mean that the operation wouldn't be reversible as it is now through addition.

No, I like how it works now.

Reply:
> I think that would be better because we could clearly check in the diff if there are any changes to the previous behavior. I would like that.

Yeah, I agree that would be better. I'll fix that when I get home later today. 


> No, I like how it works now.

I’m happy you do, I find it useful to be able to convert back and forth. I have considered making some arithmetic functions for the unit list in a future PR, which is a lot more useful if the values are accurate representations.